src/web/mjs/engine/Settings.mjs

```diff
@@ -93,6 +93,13 @@ export default class Settings extends EventTarget {
             value: false
         };
 
+        this.ignoreErrorOnDownload = {
+            label: 'Ignore errors on download',
+            description: 'Download pages ignoring errors',
```

Comment:
I recommend to use a more detailed description like `The download will be successful, even if all or some files failed to download and the chapter is in a broken state.`

Otherwise i have a bad feeling that this settings may backfire and users will start complaining about broken downloads.

Reply:
Changed it myself since OP isnt answering. I hope you dont mind